archives/2018-04-15/Overview.html

```diff
@@ -1571,13 +1567,6 @@ <h2 class="no-num no-toc no-ref heading settled" id="status"><span class="conten
 <p> This document is governed by the <a href="https://www.w3.org/2017/Process-20180201/" id="w3c_process_revision">1 February 2018 W3C Process Document</a>.</p>
 
   </div>
-  <div class="snapshot" hidden="">
-   <p>
-    This is a Recommendation Track snapshot. For the latest updates, possibly including important
- bug fixes, please see the <a href="https://w3c.github.io/webvtt/">Draft Community Group Report</a>. 
-    <button>Dismiss</button>
```

Comment:
Not sure why the snapshot guidance needs to be removed; I agree with @silviapfeiffer we need W3 guidance on what is "normal" when managing specs in two places at once like this.

Reply:
That's true. Plus I think the best forum for communicating about the spec is via GitHub repo issues, and anyone can do that whether they're in the CG, WG, both or neither.

Reply:
I propose we change
&lt;a href="https://w3c.github.io/webvtt/">Draft Community Group Report&lt;/a>
to
&lt;a href="https://w3c.github.io/webvtt/">Editor's Draft&lt;/a>

Reply:
OK by me

Reply:
There should already be an Editor's Draft link at the top of the document. If the solution is to change the link to the ED then we should just remove the whole paragraph as per the pull request. Folk don't need to be told that the ED might be more up to date, that's obvious.